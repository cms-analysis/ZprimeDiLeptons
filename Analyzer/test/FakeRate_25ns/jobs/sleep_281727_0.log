000 (281727.000.000) 08/30 22:48:26 Job submitted from host: <90.147.169.224:9618?addrs=90.147.169.224-9618&noUDP&sock=2441165_0900_3>
...
001 (281727.000.000) 08/30 22:50:27 Job executing on host: <90.147.168.80:40137>
...
006 (281727.000.000) 08/30 22:50:36 Image size of job updated: 105208
	63  -  MemoryUsage of job (MB)
	63700  -  ResidentSetSize of job (KB)
...
006 (281727.000.000) 08/30 22:50:38 Image size of job updated: 105356
	63  -  MemoryUsage of job (MB)
	63700  -  ResidentSetSize of job (KB)
...
005 (281727.000.000) 08/30 22:50:39 Job terminated.
	(1) Normal termination (return value 1)
		Usr 0 00:00:08, Sys 0 00:00:01  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 00:00:08, Sys 0 00:00:01  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	98511  -  Run Bytes Sent By Job
	69180  -  Run Bytes Received By Job
	98511  -  Total Bytes Sent By Job
	69180  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated
	   Cpus                 :                 1         1
	   Disk (KB)            :      214       75   2706010
	   Memory (MB)          :       63     2500      2500
...
