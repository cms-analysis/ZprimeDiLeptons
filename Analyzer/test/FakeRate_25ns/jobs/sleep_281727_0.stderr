mv: impossibile fare stat di `/home/condor/dir_10694/CMSSW803-Analyse_ZprimeToMuMu_13TeV.root': No such file or directory
mv: impossibile fare stat di `/home/condor/dir_10694/CMSSW803-Analyse_ZprimeToMuMu_13TeV_cand.txt': No such file or directory
